litgpt/model.py

```diff
@@ -654,13 +761,27 @@ def __init__(
         self.register_buffer("v", torch.zeros(v_shape, device=device, dtype=dtype), persistent=False)
 
     def forward(self, input_pos: torch.Tensor, k: torch.Tensor, v: torch.Tensor) -> Tuple[torch.Tensor, torch.Tensor]:
+        """
+        Writes new values `k`, `v` into cache at positions in `input_pos` along
+        the sequence dimension. Their batch size `bs` must be `<= batch_size`.
+        Returns the complete buffers, selected to batch size `bs`.
+
```

Comment:
Writes new values `k` and `v` into the cache at the positions specified by `input_pos` along the sequence dimension (`max_seq_length`). The batch size of `k` and `v` (`bs`) must be smaller or equal to KVCache batch size. Returns the full buffers, adjusted to the batch size `bs`.